joomla/joomla-cms
# [4.0] Rephrase location to file in administrator/com_installer.ini

Pull Request for Issue #33243 .

### Summary of Changes
Rephrased "Please select a package location." to ""Please select a package file." in administrator\language\en-GB\com_installer.ini


### Testing Instructions
1. Open Joomla Administrator Panel
2. Navigate to System -> Update Joomla -> Upload & Update tab
3. Click Upload & Install (without uploading any file)
Verify the text in the alert message.

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/115763957-a0b2fe00-a3c2-11eb-8266-a30db0924c1c.png)
Note: At the time of making this PR, I'm facing some issues uploading images to GitHub so in case the above image doesn't display, please use this link: https://ibb.co/PgGkHzQ 

### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/115763898-8da02e00-a3c2-11eb-9667-a2e5a0d1dc76.png)
Link if the image is broken: https://ibb.co/nLKX0sQ

### Documentation Changes Required
None
